src/odemis/model/_metadata.py

```diff
@@ -61,11 +61,32 @@
 MD_LENS_RI = "Lens refractive index"  # float (ratio), refractive index
 MD_FILTER_NAME = "Filter name"  # str, product name of the light filter
 # TODO: might need to merge DWELL_TIME and EXP_TIME into INTEGRATION_TIME: the time each pixel receive energy
+
+# extended fibsem metadata
+MD_STAGE_POSITION = "Stage position"  # dict of str -> float, position of the stage (in m/rad) for each axis
```

Comment:
I still it's too confusing with MD_POS. It should be clear that it's very rarely useful information. I'd suggest to name it `MD_STAGE_POSITION_RAW = "Stage position raw` . 
Add in the comment that it's the position *in the chamber*, using the values as read from the hardware.

For the unit, can you change the wording from "m/rad" (which sounds like a unit) to "m or rad".